docs/src/explanations/naming.md

```diff
@@ -202,6 +202,50 @@ class Example8 extends Module {
 ChiselStage.emitVerilog(new Example8)
 ```
 
+### Behavior for "Unnamed signals" (aka "Temporaries")
+
```

Comment:
nice. I was coming here to say "doesn't this deserve a docs update" and here it is!